content/en/core/releases/4.2.0.md

```diff
@@ -19,6 +19,19 @@ None.
 
 ### UI/UX changes
 
+### Floating Action Button
+
+The additive actions (creating reports, places, people, etc...) have moved from the bottom action bar to a Floating Action Button that opens a menu with all actions. This change aligns the CHT more closely with Android UX and [material design patterns](https://m3.material.io/components/menus/overview), and applies to the Messages, Reports, and Contacts tab.
+
+{{< figure src="../images/4.2.0-7998-mobile.png" link="floating-action-button-mobile.png" class="left col-3 col-lg-3" >}}
+
+{{< figure src="../images/4.2.0-7998-desktop.png" link="floating-action-button-desktop.png" class="left col-9 col-lg-9" >}}
+
+{{< figure src="../images/4.2.0-7998-desktop-open.png" link="floating-action-button-desktop.png" class="left col-9 col-lg-9" >}}
+<br clear="all">
+
+- [#7998](https://github.com/medic/cht-core/issues/7998): Move additive actions from Action Bar to Floating Action Button
+
 #### More Options menu
 
 The Export, Edit, and Delete actions have moved from the bottom action bar to a new More Option (⋮) menu located on the top right side of the screen. This change aligns the CHT more closely with Android UX and [material design patterns](https://m3.material.io/components/menus/overview), and applies to the Messages, Reports, and Contacts tab. 
```

Comment:
```suggestion
The Export, Edit, and Delete actions have moved from the bottom action bar to a new More Option (⋮) menu located on the top right side of the screen. This change aligns the CHT more closely with Android UX and [material design patterns](https://m3.material.io/components/menus/overview), and applies to the Messages, Reports, and Contacts tab. 

NOTE: The bottom action bar can be temporarily re-enabled for specific users by granting them the `can_view_old_action_bar` permission; However, it will be completely removed in a future release. See the [Feature Flags](https://docs.communityhealthtoolkit.org/apps/guides/updates/feature-flags/) documentation for more information.
```